### PR TITLE
cleanup `<weight></weight>` from docs comments

### PR DESCRIPTION
**Pull request by vivekvpandya**
Fixes https://github.com/paritytech/substrate/issues/13261

### CODE REVIEW

**Review by shawntabrizi — APPROVED** [review]

File: frame/collective/src/lib.rs [hunk 1]
```diff
@@ -361,19 +361,11 @@ pub mod pallet {
...
 		/// with other logic managing the member set.
 		///
-		/// # <weight>
-		/// ## Weight
+		/// ## Complexity:
 		/// - `O(MP + N)` where:
 		///   - `M` old-members-count (code- and governance-bounded)
 		///   - `N` new-members-count (code- and governance-bounded)
 		///   - `P` proposals-count (code-bounded)
-		/// - DB:
```
Comment (KiChjang) [inline frame/collective/src/lib.rs:370]: Huh, why is this also being removed?

Comment (shawntabrizi) [reply frame/collective/src/lib.rs:370]: DB information is now generated automatically by the benchmarking process in the weights file.

File: frame/collective/src/lib.rs [hunk 2]
```diff
@@ -425,13 +417,10 @@ pub mod pallet {
...
-		/// ## Weight
-		/// - `O(M + P)` where `M` members-count (code-bounded) and `P` complexity of dispatching
-		///   `proposal`
-		/// - DB: 1 read (codec `O(M)`) + DB access of `proposal`
-		/// - 1 event
-		/// # </weight>
+		/// ## Complexity:
+		/// - `O(B + M + P)` where:
+		/// - `B` is `proposal` size in bytes (length-fee-bounded)
+		/// - `M` members-count (code-bounded) and `P` complexity of dispatching `proposal`
```
Comment (KiChjang) [inline frame/collective/src/lib.rs:423]:
```suggestion
		/// - `M` members-count (code-bounded)
		/// - `P` complexity of dispatching `proposal`
```

File: frame/society/src/lib.rs [hunk 3]
```diff
@@ -869,15 +860,12 @@ pub mod pallet {
 		/// Parameters:
 		/// - `pos`: Position in the `Bids` vector of the bid who should be unvouched.
 		///
-		/// # <weight>
+		/// Total Complexity: O(B)
 		/// Key: B (len of bids)
 		/// - One storage read O(1) to check the signer is a vouching member.
 		/// - One storage mutate to retrieve and update the bids. O(B)
```
Comment (KiChjang) [inline frame/society/src/lib.rs:866]: Looks like this still needs to be removed? Same for the other extrinsics in this file.

**Review by KiChjang — APPROVED** [review]

File: frame/elections-phragmen/src/lib.rs [hunk 4]
```diff
@@ -479,10 +479,12 @@ pub mod pallet {
 		///   next round.
 		///
 		/// The dispatch origin of this call must be signed, and have one of the above roles.
-		///
-		/// # <weight>
 		/// The type of renouncing must be provided as witness data.
-		/// # </weight>
+		///
+		/// # Complexity
```
Comment (KiChjang) [inline frame/elections-phragmen/src/lib.rs:484]:
```suggestion
		/// ## Complexity
```

File: frame/elections-phragmen/src/lib.rs [hunk 5]
```diff
@@ -479,10 +479,12 @@ pub mod pallet {
...
+		/// 	- Renouncing::Candidate(count): O(count + log(count))
+		/// 	- Renouncing::Member: O(1)
+		/// 	- Renouncing::RunnerUp: O(1)
```
Comment (KiChjang) [inline frame/elections-phragmen/src/lib.rs:487]: CI is failing on these lines, maybe put the `Renouncing::*` variants in backticks?

Comment (vivekvpandya) [reply frame/elections-phragmen/src/lib.rs:487]: Yes checked that, fixing it.

**Comment by vivekvpandya** [comment]
@KiChjang if possible could you please add relevant labels?

**Review by kianenigma — APPROVED** [review]

**Comment by kianenigma** [comment]
Your DOT address for a tip please?

**Comment by kianenigma** [comment]
bot rebase

**Comment by vivekvpandya** [comment]
> Your DOT address for a tip please?

Thank you sir but no need for top.  if I could more complex tasks then I would like interview chance for full-time role ,😃

**Comment by vivekvpandya** [comment]
> > Your DOT address for a tip please?
> 
> Thank you sir but no need for tip. if I could finish more complex tasks then I would like interview chance for full-time role ,😃

**Comment by kianenigma** [comment]
bot rebase

**Comment by vivekvpandya** [comment]
It looks like CI will pass all checks this time. Kindly merge it.

**Comment by kianenigma** [comment]
bot merge

**Pull request merged**